docs/source/tutorials/submission.ipynb

```diff
@@ -0,0 +1,233 @@
+{
```

Comment:
<ul><li>For the note  it would be better to give the link directly to the folder within the RP git repo</li><li>batch job depend on  -&gt; batch job depends on </li><li>Same here it would be good to use "user" instead of "you"</li><li> HPC platform tht you want to use. -&gt;  HPC platform that user wants to use.</li><li> independent from the way in which -&gt;  independent of how</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2874/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>